src/main/java/duke/task/TaskManager.java

```diff
@@ -0,0 +1,140 @@
+            
+        case DEADLINE_COMMAND:
+            try {
+                String dueDate = StringManipulator.getStringAfterSlash(input);
+                String taskDescription = StringManipulator.getStringAfterWhiteSpaceAndBeforeSlash(input);
+                Deadline d = new Deadline(taskDescription,dueDate);
+                d.checkIfDeadlineDescriptionExists(taskDescription);
+                taskArrayList.add(d);
+                printMessageAfterTaskIsAdded(d);
+                updateDataFile();
+            } catch (TaskDescriptionMissingException e) {
+                System.out.println(e.getMessage());
+                MainUI.printDivider();
+            } catch (IOException e) {
+                System.out.println("IOError at DEADLINE Command");
+            }
+            break;
+        case EVENT_COMMAND:
+            try {
+                String eventPeriod = StringManipulator.getStringAfterSlash(input);
+                String taskDescription = StringManipulator.getStringAfterWhiteSpaceAndBeforeSlash(input);
+                Event e = new Event(taskDescription,eventPeriod);
+                taskArrayList.add(e);
+                printMessageAfterTaskIsAdded(e);
+                updateDataFile();
+            } catch (IOException e) {
+                System.out.println("IOError at EVENT Command");
+            }
+            break;
+        case DELETE_COMMAND:
+            try {
+                int taskNumberDeleted = StringManipulator.getTaskNumberDeleted(input);
+                System.out.println("Noted. I've removed this task:\n" + taskArrayList.get(taskNumberDeleted-1));
+                taskArrayList.remove(taskNumberDeleted-1);
+                Task.decreaseTaskCount();
+                System.out.println("Now you have " + Task.getTaskCount() + " tasks in the list.");
+                updateDataFile();
+        } catch (IOException e) {
+            System.out.println("IOError at DELETE Command");
+        }
+            break;
+        default:
+            throw new CommandNotFoundException();
+        }
+    }
+
+
+    private static void printMessageAfterTaskIsAdded(Task task) {
+        MainUI.printDivider();
+        System.out.println(ADDED_TASK_MESSAGE);
+        System.out.println("\t"+task);
+        printTaskCount();
+        MainUI.printDivider();
+    }
+
+    private static void printTaskCount() {
+        System.out.println("Now you have " + Task.getTaskCount() + " task(s) in the list.");
+    }
+
+    public static void markTaskAsDone(int taskNumber){
+        System.out.println(MARKED_TASK_AS_DONE_MESSAGE);
+        taskArrayList.get(taskNumber-1).markAsDone();
+        System.out.println(taskArrayList.get(taskNumber-1));
+        MainUI.printDivider();
+
+    }
+
+    public static void printAllTasks(ArrayList<Task> taskArrayList){
+        int TaskCount = 0;
+        MainUI.printDivider();
+        System.out.println("Here are the tasks in your list:");
+        for (Task task: taskArrayList){
```

Comment:
Can consider spacing out a bit more evenly. For example,
`for (Task task: taskArrayList) {`
Similar changes can be made in other places as well.

Reply:
Thanks, I have made the changes :)